tests/trestle/core/commands/author/profile_test.py

```diff
@@ -356,7 +359,8 @@ def test_profile_failures(tmp_trestle_dir: pathlib.Path, monkeypatch: MonkeyPatc
         required_sections='NeededExtra',
         regenerate=False,
         version=None,
-        allowed_sections=None
+        allowed_sections=None,
+        namespace=''
```

Comment:
What about the test where namespace is set to something?

Reply:
I have updated the PR description to indicate namespace is not functional and is only initially sketched in.  There is an issue describing how namespaces should work, so the matter is tracked.  This pr only addresses new functionality for display-name.